src/pages/ProductionManagement/partials/Diagrams/TR11/index.tsx

```diff
@@ -1,27 +1,27 @@
 import { useMonitoringStore } from '~/stores/useMonitoringStore';
 
-import { useFiguresCoordinateContext } from '~/pages/ProductionManagement/context/FiguresCoordinateContext';
+import { useCurrentBlueprintContext } from '~/pages/ProductionManagement/context/BlueprintContext';
 import { Figures } from '~/pages/ProductionManagement/helpers/diagrams';
 import Tr11StaticDiagram from '~/pages/ProductionManagement/partials/Diagrams/TR11/Tr11StaticDiagram';
 
 import './styles.css';
 
 export function TR11Diagram() {
-  const figuresCoordinateList = useFiguresCoordinateContext();
+  const { sensorConfigurations } = useCurrentBlueprintContext();
   const figureValues = useMonitoringStore((state) => state.figureValues);
 
   return (
     <svg
-      data-name='Layer 1'
-      id='Layer_1'
+      data-name='Layer 2'
+      id='Layer_2'
```

Comment:
Remove all id & data-name attributes from the diagram if it doesn't need

Reply:
We need the id for styling, I'll remove the data-name/